GP03FLMA/GP03FLMA_D00005_ingest.csv

```diff
@@ -1,20 +1,20 @@
-#,,GP03FLMA-RIM01-02-CTDMOG041,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
-#,,GP03FLMA-RIM01-02-CTDMOG042,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
-#,,GP03FLMA-RIM01-02-CTDMOG043,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
-#,,GP03FLMA-RIM01-02-CTDMOG044,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
-#,,GP03FLMA-RIM01-02-CTDMOG045,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
-#,,GP03FLMA-RIM01-02-CTDMOG046,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
-#,,GP03FLMA-RIM01-02-CTDMOG047,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
-#,,GP03FLMA-RIM01-02-CTDMOG048,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
-#,,GP03FLMA-RIM01-02-CTDMOH049,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
-#,,GP03FLMA-RIM01-02-CTDMOH050,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
-#,,GP03FLMA-RIM01-02-CTDMOH051,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
-#mi.dataset.driver.sio_eng.sio.sio_eng_sio_telemetered_driver,,GP03FLMA-RIS01-00-SIOENG000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
-#mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_telemetered_driver,,GP03FLMA-RIS01-03-DOSTAD000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
-#mi.dataset.driver.phsen_abcdef.sio.phsen_abcdef_sio_telemetered_driver,,GP03FLMA-RIS01-04-PHSENF000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
-#mi.dataset.driver.flort_dj.sio.flort_dj_sio_telemetered_driver,,GP03FLMA-RIS01-05-FLORTD000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.sio_eng.sio.sio_eng_sio_telemetered_driver,/omc_data/whoi/OMC/GP03FLMA/D00005/*status_1328401.dat,GP03FLMA-RIM01-00-SIOENG000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)

```

Comment:
I agree with Sheri! Reviewed persistor serial numbers with James and Sheri is correct 12365